src/identity/components/GlobalHeader/AccountDropdown.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {FC} from 'react'
+import {IconFont} from '@influxdata/clockface'
+import {MenuDropdown, SubMenuItem} from '@influxdata/clockface'
+import {OrganizationSummaries, UserAccount} from 'src/client/unityRoutes'
+
+type OrgSummaryItem = OrganizationSummaries[number]
+
+interface Props {
+  activeOrg: OrgSummaryItem
+  activeAccount: UserAccount
+  // setActiveAccount: Function
+  accountsList: UserAccount[]
+}
+
+export const AccountDropdown: FC<Props> = ({
+  activeOrg,
+  activeAccount,
+  accountsList,
+}) => {
+  const selectedAccount = {
+    id: activeAccount.id.toString(),
+    name: activeAccount.name,
+  }
+
+  const accountMainMenu = [
+    {
+      name: 'Settings',
+      iconFont: IconFont.CogOutline,
+      href: `/orgs/${activeOrg.id}/accounts/settings`,
+    },
+    // 'Members' is listed on the Figma, but discussed with design, and this may not exist yet.
+    // {
+    //   name: 'Members',
```

Comment:
Flagging this because it's on the figma, but I don't believe the page exists presently. 

@juliajames12 - Did I get it right from our conversation that we don't need to redirect to an accounts 'Members' page at this time?

Reply:
Agree. Based on the conversation with design, this link isn't needed at this time.